cEP-0029.md

```diff
@@ -0,0 +1,63 @@
+# Gitmate for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 29                                                    |
+| Version  | 1.0                                                   |
+| Title    | Gitmate for coala                                     |
+| Authors  | Sanchit Gupta <mailto:sanchitgupta.072@gmail.com>     |
+| Status   | Implementation Due                                    |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to IGitt and Gitmate, which include
```

Comment:
```suggestion
This cEP describes the improvements to IGitt and Gitmate, which includes
```

Reply:
I think this will be 'include' as it is preceded by 'improvements' :)